templates/filter/filter-guidelines.html.twig

```diff
@@ -0,0 +1,29 @@
+{#
+/**
+ * @file
+ * Theme override for guidelines for a text format.
+ *
+ * Available variables:
+ * - format: Contains information about the current text format, including the
+ *   following:
+ *   - name: The name of the text format, potentially unsafe and needs to be
+ *     escaped.
+ *   - format: The machine name of the text format, e.g. 'basic_html'.
+ * - attributes: HTML attributes for the containing element.
+ * - tips: Descriptions and a CSS ID in the form of 'module-name/filter-id'
+ *   (only used when 'long' is TRUE) for each filter in one or more text
+ *   formats.
+ *
+ * @see template_preprocess_filter_tips()
+ */
+#}
+{%
+  set classes = [
+    'filter-guidelines__item',
```

Comment:
Why are we adding classes that seem very similar to what has been added in the preprocess function?

Reply:
See the comment on the preprocess function:  `// Fix filter guidelines selector issue of 'filter/drupal.filter'`

Right now that `js` bases on `filter-guidelines-item` and  `filter-guidelines-[format_id]`classes.

(I forgot to report the bug on dorg.)

Reply:
@lauriii just added todo comment with the dorg issue link